SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewControllerDelegate.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SKYChatParticipantListViewControllerDelegate.swift
+//  SKYKitChat
+//
+//  Copyright 2016 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+@objc public protocol SKYChatParticipantListViewControllerDelegate: class {
```

Comment:
> My recommendation is to keep the directory structure flat (no need for `ParticipantList` subdirectory)

I think the structure will be clearer when there are more and more UI components are coming, e.g. the custom `UITableViewCell` and the custom `UIButton`, etc.

> This delegate can be defined in the `SKYChatParticipantListViewController.swift`

Okay. Let me merge this two files.